### PR TITLE
1h00min to 1h

### PR DESCRIPTION
**Pull request by CuauhFrMx**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] You can **build successfully** the whole project with your changes locally
- [x] You use the coding conventions and the **naming conventions** of the project
- [x] You use **tabs** for indentation
- [x] You make sure that your PR is **not a duplicate**
- [x] This PR is **ready** to be reviewed and merged
- [x] This PR merges into the **`development`** branch
- [x] There is no **`TODO`** in the code
- [x] There is no spelling or grammatical errors in the code
- [x] Details of the issue or feature are documented below
- [x] This PR is **not** a **breaking change** (e.g. changes that would cause existing functionality to change)

## Proposed changelog

| Français `fr` |
| --- |
| Deux coquilles, et changement de "1h00min" à "1h" pour la durée d'un cours |

### CODE REVIEW

File: src/views/HomepageView.vue [hunk 1]
```diff
@@ -556,7 +556,7 @@
 								<h2>{{ grade.grade.value }}<small>/{{ grade.grade.out_of }}</small></h2>
 							</ion-label>
 							<ion-label v-else>
-								<h2>{{ grade.info.significantReason }}<small>/{{ grade.grade.out_of }}</small></h2>
```
Comment (lucas-luchack) [inline src/views/HomepageView.vue:559]: Ceci ne doit pas être modifié, l'affichage du total de la note doit apparaître.

**Review by lucas-luchack — CHANGES_REQUESTED** [review]
Un truc d'incorrect, le reste c'est bon

File: src/views/HomepageView.vue (hunk 1 above)
Comment (lucas-luchack) [reply src/views/HomepageView.vue:559]: Je n'ai rien dis, je n'avais pas vu que c'était sur la Homepage

**Review by lucas-luchack — APPROVED** [review]

**Comment by CuauhFrMx** [comment]
nickel merci, oui en effet sur la homepage c'est pas nécéssaire je pense

**Pull request merged**